mpv_test.go

```diff
@@ -0,0 +1,23 @@
+package mpv
+
+import (
+	"testing"
+)
+
+func TestCreate(t *testing.T) {
+	mpv := Create()
+	if mpv == nil {
+		t.Fatal("Create() returned nil")
+	}
+}
+
+func TestClientName(t *testing.T) {
+	mpv := Create()
+	if err := mpv.Initialize(); err != nil {
+		t.Fatalf("Initialize() error: %v", err)
+	}
+	name := mpv.ClientName()
```

Comment:
I'm pretty sure this returning an empty string is expected unless you set the client name like https://github.com/dweymouth/supersonic/blob/main/backend/player/mpv/player.go#L124

(e.g. I think either getting rid of the client name portion of the test, or setting a client name before getting it and checking that it is the same)

Reply:
For me the `name` is `main`. I didn't think much about it since this is basically just a compile/linker test for testing mpv's library and include path.

```go
func TestClientName(t *testing.T) {
	mpv := Create()
	if err := mpv.Initialize(); err != nil {
		t.Fatalf("Initialize() error: %v", err)
	}
	name := mpv.ClientName()
	t.Fatalf("its (%s)", name)
	if name == "" {
		t.Fatal("ClientName() returned an empty string")
	}
}
```
leads to:
```text
--- FAIL: TestClientName (0.03s)
    /home/pb/Projects/go-mpv/mpv_test.go:20: its (main)
```